cl/users/email_handlers.py

```diff
@@ -143,6 +145,10 @@ def handle_soft_bounce(
                                 "notification_subtype": EMAIL_NOTIFICATIONS.MAX_RETRY_REACHED,
                             },
                         )
+                        # After ban email address, delete Backoff Event
```

Comment:
Is it worth providing more detail here about why? Maybe reference this PR or explain it briefly in the code?

Reply:
Sure, added more details and reference to this PR as well.